.github/workflows/clang-format.yml

```diff
@@ -4,16 +4,17 @@ on: [push, pull_request]
 
 jobs:
   check:
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-latest
     env:
       CLANG_VERSION: 10
     steps:
     - uses: actions/checkout@v2
     - name: Install clang-format
       run: |
+        codename=$( lsb_release -cs )
```

Comment:
I generally prefer to use long option names in scripts, but it's not a blocker.

Reply:
I think that sounds good. Updated.